common/network_util.cpp

```diff
@@ -51,4 +51,14 @@ const std::string& getLocalIPAddress() {
   return local_ip;
 }
 
+std::string getNetworkAddressStr(const folly::SocketAddress& addr) noexcept {
+  std::string add_str = "unknown_addr";
```

Comment:
In your description, I saw a few entries without an upstream address. When would that happen?

Reply:
yeah, I noticed that, I it's all for MASTER/Leader replica, the upstream should really be localhost, but i think we don't set that --- I'll look into that separately.